_drafts/SC2-1-2-no-keyboard-trap-non-standard-navigation.md

```diff
@@ -0,0 +1,153 @@
+---
+name: No keyboard trap non-standard navigation
+group:
+- SC2-1-2-no-keyboard-trap-standard-navigation
+- SC2-1-2-no-keyboard-trap-non-standard-navigation (current)
+
+description: |
+  This rule checks if the user is advised on a method for non-standard keyboard navigation to navigate through focusable content on a web page without becoming trapped in any element.
+
+success_criterion: 
+- 2.1.2 # No Keyboard Trap
+
+test aspects:
+- DOM Tree
+
+authors:
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+- Malin Øvrebø
+- Shadi Abou-Zahra
+- Carlos Duarte
+- Anne Thyme Nørregaard
+- Stein Erik Skotkjerra
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element on a web page that is [focusable][] and reachable through [sequential focus navigation](https://www.w3.org/TR/html/editing.html#sequential-focus-navigation) where focus cannot cycle to the browser UI by using [standard keyboard navigation][].
```

Comment:
Remove "and reachable through [sequential focus navigation](https://www.w3.org/TR/html/editing.html#sequential-focus-navigation)"